File: iteration1/src/controllers/UserController.java
package iteration1.src.controllers;

import iteration1.src.models.User;
import iteration1.src.utils.AuthenticateService;

public class UserController {

    private User currentUser;

    public UserController() {
        currentUser = null;
    }

    public User loginUser(String username, String password) {
        AuthenticateService authenticateService = new AuthenticateService();
        return currentUser = authenticateService.authenticateUser(username, password);
    }
/*
    void logoutUser() {
        currentUser.logout();
        currentUser = null;
    }
*/
    User getCurrentUser() {
        return currentUser;
    }

    void setCurrentUser(User currentUser) {
        this.currentUser = currentUser;
    }

}

File: iteration3/src/controllers/UserController.py
from iteration3.src.utils.AuthenticateService import AuthenticateService
from iteration3.src.models.User import User

class UserController:
    def __init__(self):
        self._currentUser = None

    def login(self, userId, password)->User:
        authenticateService = AuthenticateService()
        self._currentUser = authenticateService.authenticateUser(userId, password)
        return self._currentUser


    